frontend/src/components/eventPage/DisplayEventComments.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+const DisplayEventComments = (comments) => {
+  const [form, setForm] = useState(false);
+  const [userComment, setUserComment] = useState("");
+
+  let displayComments = comments.comments;
+  let Logged = false;
+  const url = import.meta.env.VITE_URL;
+  const current_link = window.location.pathname;
+  const current_eventId = current_link.split("/")[2];
+  let user = localStorage.getItem("user");
+  let jsonUser = JSON.parse(user);
+
+  if (!(user.includes(null) || user.includes("null"))) {
+    Logged = true;
+  }
+
```

Comment:
All the pages' logged in status should depend on user state, not cookie. Cookie comes in when user state is null.
Consider this case, a lot of us don't like cookies these days because they can take up device memory and can increase the risk of personal info leaking. So can I log in and use the website still if I don't give permission to create cookies? Take a look at facebook or amazon. If you don't allow cookies, it only means that every time when you close the browser and reopen it, you need to sign in again. Event Finder right now does not allow log in if the user doesn't allow cookies.

Reply:
**_To Do:_**
Re-work user state & cookies